torch/nn/utils/parametrize.py

```diff
@@ -188,7 +184,7 @@ def register_parametrization(
     When accessing ``module[tensor_name]``, the module will return the
     parametrized version ``parametrization(module[tensor_name])``. The backward
     pass will differentiate through the ``parametrization`` and if the original
-    tensor is a :class:``~Parameter``, it will be updated accordingly by the optimizer.
+    tensor is a :class:``torch.nn.Parameter``, it will be updated accordingly by the optimizer.
```

Comment:
This is supposed to have a single backtick I think. It does not render properly: https://12180955-65600975-gh.circle-artifacts.com/0/docs/generated/torch.nn.utils.parametrize.register_parametrization.html?highlight=register_parametrization#torch.nn.utils.parametrize.register_parametrization